### PR TITLE
Various fixes for omega

### PR DESCRIPTION
**Pull request by jflat06**
A couple issues I've noticed with the omega implementation.

### CODE REVIEW

File: tmol/score/common/uaid_util.hh [hunk 1]
```diff
@@ -26,7 +26,6 @@ TMOL_DEVICE_FUNC int resolve_atom_from_uaid(
     TView<Int, 2, D> pose_stack_block_coord_offset,
     TView<Int, 2, D> pose_stack_block_type,
     TView<Vec<Int, 2>, 3, D> pose_stack_inter_block_connections,
-    TView<Int, 3, D> block_type_omega_quad_uaids,
```
Comment (jflat06) [inline tmol/score/common/uaid_util.hh:29]: Unnecessary argument.

File: tmol/score/omega/omega_energy_term.py [hunk 2]
```diff
@@ -38,10 +38,10 @@ def setup_block_type(self, block_type: RefinedResidueType):
         if hasattr(block_type, "omega_quad_uaids"):
             return
 
-        omega_quad_uaids = numpy.full((4, 3), -1, dtype=numpy.float32)
+        omega_quad_uaids = numpy.full((4, 3), -1, dtype=numpy.int32)
         if "omega" in block_type.torsion_to_uaids:
             omega_quad_uaids = numpy.array(
-                block_type.torsion_to_uaids["omega"], dtype=numpy.float32
+                block_type.torsion_to_uaids["omega"], dtype=numpy.int32
```
Comment (jflat06) [inline tmol/score/omega/omega_energy_term.py:44]: Wrong data types.

File: tmol/tests/score/omega/test_omega_energy_term.py [hunk 3]
```diff
@@ -30,7 +30,7 @@ def test_annotate_omega_uaids(ubq_res, default_database, torch_device: torch.dev
     assert pbt.omega_quad_uaids.device == torch_device
 
 
-def test_whole_pose_scoring_module_gradcheck_whole_pose(
+def test_whole_pose_scoring_module_gradcheck(
```
Comment (jflat06) [inline tmol/tests/score/omega/test_omega_energy_term.py:33]: Wrong name.

File: tmol/score/omega/omega_energy_term.py (hunk 2 above)
Comment (aleaverfay) [reply tmol/score/omega/omega_energy_term.py:44]: Ah, but at least you previously converted to int32 in setup_packed_block_types. (I had to figure out why you didn't also have to change the data type on the C++ side after making this float->int change)

**Review by aleaverfay — APPROVED** [review]
LGTM

**Pull request merged**